test/IntegrationTestCommon/Factories/WebApplicationFactoryBase.cs

```diff
@@ -145,7 +145,10 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 // Email Verification
                 { "globalSettings:enableEmailVerification", "true" },
                 { "globalSettings:disableUserRegistration", "false" },
-                { "globalSettings:launchDarkly:flagValues:email-verification", "true" }
+                { "globalSettings:launchDarkly:flagValues:email-verification", "true" },
+
+                // New Device Verification
+                { "globalSettings:disableEmailNewDevice", "false" },
```

Comment:
❓ : I didn't realize this PR was going to be responsible for introducing this flag and modifying the logic?

Reply:
This existed before in the `BaseRequestValidator` I added it here to facilitate testing.

[on main](https://github.com/bitwarden/server/blob/main/src/Identity/IdentityServer/BaseRequestValidator.cs#L569)

Reply:
Ah, I'm sorry. I read the comment and assumed it was newly added.